artifact-version-diff/src/main/resources/3.15_allowed_artifacts.yaml

```diff
@@ -0,0 +1,73 @@
+allowed-artifacts-version-comparison:
+  # https://issues.redhat.com/browse/QUARKUS-5043
+  - artifact:
+      com.google.guava:guava
+    allowed-rhbq-versions:
+      - 33.0.0.jre-redhat-00002
+  # https://issues.redhat.com/browse/QUARKUS-5063?focusedId=25890973&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25890973
+  - artifact:
+      org.hibernate.common:hibernate-commons-annotations
+    allowed-rhbq-versions:
+      - 7.0.3.Final-redhat-00001
+  # https://issues.redhat.com/browse/QUARKUS-4089
+  - artifact:
+      org.slf4j:slf4j-api
+    allowed-rhbq-versions:
+      - 2.0.7.redhat-00003
+  # all elytron https://issues.redhat.com/browse/QUARKUS-5226
```

Comment:
Note:
I split it by public and productized dependencies
Otherwise if we want have it as one we need to set version to `2.5.1.Final` and not productized. 

Something like this
```
- artifact:
    org.wildfly.security:wildfly-elytron*
  allowed-rhbq-versions:
    - 2.5.1.Final
 ```